src/components/Tipseen/Tipseen.tsx

```diff
@@ -1,46 +1,83 @@
-import { forwardRef, Fragment, useMemo, useRef } from "react";
-import PropTypes from "prop-types";
+import { forwardRef, Fragment, ReactElement, useMemo, useRef } from "react";
+import { DialogPosition } from "../../constants/positions";
 import cx from "classnames";
 import useMergeRefs from "../../hooks/useMergeRefs";
 import Tooltip from "../../components/Tooltip/Tooltip";
 import Button from "../../components/Button/Button";
 import { BEMClass } from "../../helpers/bem-helper";
 import Icon from "../../components/Icon/Icon";
 import CloseSmall from "../../components/Icon/Icons/components/CloseSmall";
-import { DialogPosition } from "../../constants/positions";
-import { AnimationType, JustifyType } from "../../constants/dialog";
+import { AnimationType, EMPTY_ARR, HideShowEvent, JustifyType } from "../../constants";
 import TipseenTitle from "./TipseenTitle";
 import { TIPSEEN_CLOSE_BUTTON_ARIA_LABEL } from "./TipseenConstants";
+import { VibeComponent, VibeComponentProps } from "../../types";
+import { MoveBy } from "../../types/MoveBy";
+import { ElementContent } from "../../types/ElementContent";
+import { Modifier } from "react-popper";
 import styles from "./Tipseen.module.scss";
 
 const TIPSEEN_BASE_CSS_CLASS = "monday-style-tipseen";
 const bemHelper = BEMClass(TIPSEEN_BASE_CSS_CLASS);
 
-const Tipseen = forwardRef(
+interface TipseenProps extends VibeComponentProps {
+  position?: DialogPosition;
+  animationType?: AnimationType;
+  hideDelay?: number;
+  showDelay?: number;
+  title?: string;
+  isCloseButtonHidden?: boolean;
```

Comment:
if we already on it let's add prop which called hideCloseButton which will do the same as this and set this as backward compatibility? (we are trying to reduce usage of is prefix on props)